.python_package/setup.cfg

```diff
@@ -0,0 +1,44 @@
+[metadata]
+name = gridtools-cpp
+author = ETH Zurich
+author_email = gridtools@cscs.ch
+license_files = LICENSE
+long_description = file: README.md
+long_description_content_type = text/markdown; charset=UTF-8
+url = https://gridtools.github.io
+project_urls = 
+	Source Code = https://github.com/GridTools/gridtools
+platforms = any
+version = 2.2.1a2
```

Comment:
Same as above, I  would like to add a comment here to document the fact that the version is actually updated from the main gridtools version file... (but configparser would delete the comment later :sweat:   )

Reply:
We can solve these by generating `setup.cfg` from `setup.cfg.in`, which would have the comments. Only question is whether `setup.cfg` should still be versioned (since it has tool config as well).